hsctl

```diff
@@ -4,15 +4,20 @@
 # HydroShare Control Script
 # Author: Michael Stealey <michael.j.stealey@gmail.com>
 
+machine="`uname -s`"
+case "$machine" in
+  Linux*)  export SED_EXT=''   ;;
+  Darwin*) export SED_EXT='""' ;;
+  *)       export SED_EXT=''   ;;
+esac
+
```

Comment:
what about Windows? the CUAHSI standard right now is Min GW (the Git for Windows shell)

Reply:
If it's too hard to do "all 3" in a single process, we can have two entirely separate setups for windows vs everything else

Reply:
Linux is Linux
Mac is Darwin
Windows is * 
=))